docs/data/material/migration/upgrade-to-v7/upgrade-to-v7.md

```diff
@@ -14,6 +14,62 @@ In the `package.json` file, change the package version from `latest` to `next`.
 Using `next` ensures your project always uses the latest v7 pre-releases.
 Alternatively, you can also target and fix it to a specific version, for example, `7.0.0-alpha.0`.
 
+## Supported browsers and versions
+
+### Minimum React version
+
+The minimum supported version of React is v17.0.0 (the same as v5 and v6).
+Use the snippet below to update your project (replace the `<version>` with the one you want):
+
+<codeblock storageKey="package-manager">
+
+```bash npm
+npm install react@<version> react-dom@<version>
+```
+
+```bash pnpm
+pnpm add react@<version> react-dom@<version>
+```
+
+```bash yarn
+yarn add react@<version> react-dom@<version>
+```
+
+</codeblock>
```

Comment:
If it's the same as v6, I think we can remove this section

Reply:
I still think that this section is useful. If I am upgrading to v7, I want to know this even though the minimum version is the same.

So I simplified this section instead.